scripts/create_source_tarball.sh

```diff
@@ -31,7 +31,6 @@ REPO_ROOT="$(dirname "$0")"/..
     fi
     # Add dependencies
     mkdir -p "$SOLDIR/deps/downloads/" 2>/dev/null || true
-    wget -O "$SOLDIR/deps/downloads/jsoncpp-1.9.3.tar.gz" https://github.com/open-source-parsers/jsoncpp/archive/1.9.3.tar.gz
```

Comment:
Why does this needs jsoncpp, but not range-v3?

`scripts/release_ppa.sh` has both.

Reply:
Because it does not need either so it did not fail when we were adding `ranges-v3` and forgot to add it here.

If it's here though, the package gets included in sources and build does not rely on an internet connection. It might have been more important in the past when our cmake config was not downloading jsoncpp automatically - this script was created in #1542 and had the wget line from the very beginning while `jsoncpp.cmake` that downloads jsoncpp was only added in #2926.

Personally, I think that external libs should not be a part of our source package but if we do want that then both libs should be listed here. In that case we should also add a cmake option to disable the download - so that we get an error here if we add more libs in the future and forget to list one of them.